models/base/scratch/default/snowplow_web_base_events_this_run.sql

```diff
@@ -143,6 +143,7 @@ with events_this_run AS (
     a.event_version,
     a.event_fingerprint,
     a.true_tstamp,
+    a.load_tstamp,
```

Comment:
Can we safely add this?

Reply:
RBD loader 4.0.0 becomes a pre-requisite for this version so in theory yes

Reply:
@agnessnowplow Where do we document this?